src/main/webapp/intrigue-api/metacards/metacards.js

```diff
@@ -116,8 +116,8 @@ const typeDefs = `
   }
 
   extend type Mutation {
-    createMetacard(attrs: MetacardAttributesInput!): MetacardAttributes
-    saveMetacard(id: ID!, attrs: MetacardAttributesInput!): MetacardAttributes
+    createMetacard(attrs: MetacardAttributesInput!, tag: String): MetacardAttributes
```

Comment:
Is the info already part of the attrs?

Reply:
The SearchFormsApplication doesn't do this currently. We could have the graphql resolver inject the attr, but I'm not sure if that's worth it or not?

Reply:
I think the resolver should inject it because it is more consistent with how other metacards are returned. Then on the way back out the info is already part of the metacard.